pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -318,3 +336,37 @@ func (r *ReconcileVirtualMachineImport) createProvider(vmi *v2vv1alpha1.VirtualM
 
 	return nil, fmt.Errorf("Invalid source type. only Ovirt type is supported")
 }
+
+func (r *ReconcileVirtualMachineImport) upsertStatusConditions(vmiName types.NamespacedName, newConditions []v2vv1alpha1.VirtualMachineImportCondition) error {
+	var instance v2vv1alpha1.VirtualMachineImport
+	err := r.client.Get(context.TODO(), vmiName, &instance)
```

Comment:
@pkliczewski Isn't there an API call to fetch only the Status of the resource instead of the entire element or did we do some custom code for that in java-kubevirt-client ?

Reply:
There is only [StatusClient](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/client/interfaces.go#L86) interface but it to not provide `Get` nor `List`